crates/tabby-common/src/api/code.rs

```diff
@@ -58,7 +58,7 @@ pub enum CodeSearchError {
 
 #[derive(Deserialize, ToSchema)]
 pub struct CodeSearchQuery {
-    pub git_url: String,
+    pub source_id: String,
```

Comment:
```suggestion
    pub git_url: String,
```

Given that we're also passing `git_url` from frontend (in `answer.rs`), we can keep the field name for simplicity.

Reply:
ok